File: src/main/java/io/github/mg138/tsbook/utils/config/gui/armor/ArmorSetting.java
package io.github.mg138.tsbook.utils.config.gui.armor;

import org.bukkit.configuration.ConfigurationSection;

public class ArmorSetting {
    public final String type;

    public ArmorSetting(ConfigurationSection section) {
        this.type = section.getString("TYPE");
    }
}

File: src/main/java/io/github/mg138/tsbook/setting/gui/armor/element/ArmorSetting.kt
package io.github.mg138.tsbook.setting.gui.element;

import org.bukkit.configuration.ConfigurationSection;

public class ArmorElementSetting {
    public final String type;

    public ArmorElementSetting(ConfigurationSection section) {
        this.type = section.getString("TYPE");
    }
}
